docs are included in check and install even if skipped

--- a/Makefile
+++ b/Makefile
@@ -107,8 +107,7 @@
 	@utils/check_license/check-headers.sh $(TOP) BSD-3-Clause
 	@echo Done.
 
-check-doc: doc
-	BUILD_RPMEM="$(BUILD_RPMEM)" utils/check-manpages
+check-doc:
 
 sparse:
 	$(MAKE) -C src sparse
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -287,15 +287,7 @@
 	$(RM) -r $(MANPAGES_WEBDIR_LINUX) \
 		$(MANPAGES_WEBDIR_WINDOWS)
 
-install: compress
-	install -d -v $(MANPAGES_DESTDIR_7)
-	install -p -m 0644 $(GZFILES_7) $(MANPAGES_DESTDIR_7)
-	install -d -v $(MANPAGES_DESTDIR_5)
-	install -p -m 0644 $(GZFILES_5) $(MANPAGES_DESTDIR_5)
-	install -d -v $(MANPAGES_DESTDIR_3)
-	install -p -m 0644 $(GZFILES_3) $(GZFILES_3_DUMMY) $(MANPAGES_DESTDIR_3)
-	install -d -v $(MANPAGES_DESTDIR_1)
-	install -p -m 0644 $(GZFILES_1) $(MANPAGES_DESTDIR_1)
+install:
 
 uninstall:
 	$(foreach f, $(notdir $(GZFILES_7)), $(RM) $(MANPAGES_DESTDIR_7)/$(f))
